### PR TITLE
Homepage.

### PR DESCRIPTION
**Pull request by YuriGizra**
#195
Output:
selection_336

### CODE REVIEW

**Comment by YuriGizra** [comment]
Added our works :
selection_340

**Comment by YuriGizra** [comment]
Imtarmediate state: 
selection_342

**Comment by YuriGizra** [comment]
Carousel markup:
selection_344

**Comment by YuriGizra** [comment]
add quote marks:
selection_346

About us section pixel perfect:
selection_345

**Comment by amitaibu** [comment]
:+1:

**Comment by YuriGizra** [comment]
Add blog section:

**Comment by amitaibu** [comment]
Image is missing

**Comment by YuriGizra** [comment]
Footer:
selection_348

**Comment by YuriGizra** [comment]
@amitaibu, ready for review.

**Comment by amitaibu** [comment]
Please update visual regression test, so it's green

**Comment by YuriGizra** [comment]
@amitaibu  Removed the Visual regression.

**Comment by amitaibu** [comment]
Can you add layered output images please

**Comment by YuriGizra** [comment]
Header layered:
selection_349
output:
selection_350

responces:
selection_351

selection_353

About us:
selection_354

selection_355

from blog:
selection_356

selection_357

Footer:
selection_358

selection_359

**Comment by bricel** [comment]
Please add fax number to footer as well.

**Comment by amitaibu** [comment]
@bricel we decided not to add the fax on account that it looks "old"

**Comment by bricel** [comment]
What the fax ! :-) It's actually still useful, we could maybe hide it somehow ?

**Comment by YuriGizra** [comment]
@amitaibu, @bricel,
So, should I add fax number  or no?

**Comment by amitaibu** [comment]
Not needed

**Comment by YuriGizra** [comment]
ok. It's ready for review.

**Comment by amitaibu** [comment]
Needs more work :)

your_project__the_gizra_way

**Comment by amitaibu** [comment]
your_project__the_gizra_way-3

**Comment by YuriGizra** [comment]
* [ ] 1200 px container width for all screen  except mobile (<767px).
* [ ] Fonts.
* [ ] Add Logo Acquila.
* [ ] header / footer  backgroun min-width 1200px.
* [ ] Add links in blog sections.

**Comment by YuriGizra** [comment]
Added font Lato. It is closest to GillSans
Fixed fonts:
added links to blog section.

selection_362

selection_363

selection_360

**Comment by amitaibu** [comment]
:+1:

**Comment by YuriGizra** [comment]
Fixed container size for all screen except mobile:
Output:
selection_364

Very high resolution screen:
selection_365

selection_366

**Comment by YuriGizra** [comment]
Added logo. Fixed according design:
selection_379
selection_380

selection_381

selection_382

**Comment by YuriGizra** [comment]
Added popup block.
acquia

**Comment by amitaibu** [comment]
Can you update the READMR with the installation needed (`gem update --system && gem install scss-lint`)

Also, I'm getting this error on `grunt serve`

```
    loading tasks  1s  ▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇ 99%
    Total 1s
        Warning: Running "bootlint:files" (bootlint) task
    Warning: Object #<Object> has no method 'forEach' Use --force to continue.
    
    Aborted due to warnings.
```

**Comment by YuriGizra** [comment]
@amitaibu, Ready for review.

File: package.json [hunk 1]
```diff
@@ -27,7 +27,8 @@
     "grunt-usemin": "~2.0.0",
     "jshint-stylish": "~0.1.3",
     "load-grunt-tasks": "~0.2.0",
-    "time-grunt": "~0.2.8"
+    "time-grunt": "~0.2.8",
+    "grunt-scss-lint": "~0.3.8"
```
Comment (amitaibu) [inline package.json]: is this still used?

File: .scss-lint.yml [hunk 2]
```diff
@@ -0,0 +1,31 @@
+# Configuration scss-lint for Gizra.com.
+
+scss_files: "app/_scss/*.scss"
```
Comment (amitaibu) [inline .scss-lint.yml]: is this still used? Maybe worth to move to own PR.

File: package.json (hunk 1 above)
Comment (YuriGizra) [inline package.json]: Yes, scss-lint is working.

**Comment by YuriGizra** [comment]
Added hover intent:
[link to video](http://gfycat.com/FrequentQuarrelsomeAmericanbittern)

Added links to projects:
* [harvard](http://openscholar.harvard.edu)
* [acquia](http://www.acquia.com)
* [commerce-guys](http://commerceguys.com)
* [european commision: c4d ](http://capacity4dev.ec.europa.eu)

@amitaibu, What link will be right for EC?
* [grammy](http://www.grammy.com)
* [wix](http://www.wix.com)
* [whitehouse](http://www.whitehouse.gov)

**Comment by amitaibu** [comment]
We will add the links later - for now it can be a placeholder

**Comment by amitaibu** [comment]
@YuriGizra please assign to me when ready for review. Also, please update title, and add a screenshot of the banner, to see the background is in place.

**Comment by YuriGizra** [comment]
Fixed background header:
selection_387

**Comment by YuriGizra** [comment]
Accidentally closed.

**Comment by YuriGizra** [comment]
jekyll recent post:
selection_389

@amitaibu, 
Should inner links  be the same color? 
selection_388

**Comment by amitaibu** [comment]
Lets take the first 300 words, and strip tags (i.e. no images, and no links)

**Comment by YuriGizra** [comment]
Stripped and  truncated:
selection_390

Centered hover [block](http://www.gfycat.com/FirstNeighboringAntlion)

**Comment by YuriGizra** [comment]
Fixed title font:
selection_404

**Comment by YuriGizra** [comment]
@amitaibu,
Ready for review.

**Comment by amitaibu** [comment]
@YaronMiro please review

File: app/index.html [hunk 3]
```diff
@@ -5,84 +5,156 @@
...
-          <li><a href="/#work">Work</a></li>
-          <li><a href="/#about">About</a></li>
-          <li><a href="/blog">Blog</a></li>
-          <li><a href="/team">Team</a></li>
-          <li><a href="/#contact">Contact</a></li>
-        </ul>
+    <div class="row">
+      <div class="col-sm-12">
+        <nav id="nav" role="navigation" class="navbar navbar-default">
+          <div class="container-fluid">
```
Comment (YaronMiro) [inline app/index.html]: @YuriGizra 
Is the `container-fluid` class really needed ?

File: app/index.html [hunk 4]
```diff
@@ -5,84 +5,156 @@
...
+              <ul class="nav navbar-nav">
+                <li><a href="/team">Meet the team</a></li>
+                <li><a href="/blog">Blog</a></li>
+                <li><a href="/#contact">Contact</a></li>
+              </ul>
+            </div>
+          </div>
+        </nav>
+        <div id="slogan">
+          <div class="gizra-logo"><a href="/">gizra</a></div>
```
Comment (YaronMiro) [inline app/index.html]: Please replace (Better for seo)
`<div class="gizra-logo">` => `<h1 class="gizra-logo">`

File: app/index.html [hunk 5]
```diff
@@ -90,34 +162,63 @@ <h2 class="title abril-font">You might have seen our work already</h2>
...
-        </div>
-      </address>
+<section id="blog">
+  <div class="container">
+    <div class="row text-wrapper">
+      <div class="col-sm-10 col-sm-offset-1">
+        <h2 class="">From our blog:</h2>
+        {% for post in site.posts limit:1 %}
+          {% if post.author == null %}
+          {% assign author = "Amitaibu" %}
```
Comment (YaronMiro) [inline app/index.html]: @YuriGizra 
Please indent so it is easy to read.
```
{% if post.author == null %}
     {% assign author = "Amitaibu" %}
{% else %}
     {% assign author = post.author %}
 {% endif %}
```
to the same on the rest of the jekyll code

Comment (YaronMiro) [inline app/index.html]: I will fix this ^^ :)

File: app/index.html (hunk 4 above)
Comment (YaronMiro) [inline app/index.html]: I will fix this ^^ :)

**Comment by YaronMiro** [comment]
Thanks Merged manually.
